cms/cms_toolbars.py

```diff
@@ -141,6 +142,16 @@ def add_admin_menu(self):
             self._admin_menu.add_modal_item(_('User settings'), url=admin_reverse('cms_usersettings_change'))
             self._admin_menu.add_break(USER_SETTINGS_BREAK)
 
+            # clipboard
```

Comment:
Documentation is always great. Have you changed anything that goes here: https://django-cms.readthedocs.org/en/latest/reference/api_references.html#cms-toolbar?